cv.py

```diff
@@ -19,8 +19,9 @@ def __init__(self):
         self.SCREEN_HEIGHT, self.SCREEN_WIDTH = frame.shape[:2]
 
         self.FIELD_OF_VIEW_RAD = 70.42 * math.pi / 180.0
-        self.DIST_CONSTANT = 3300 # need to test this
+        self.DIST_CONSTANT = 3300 # dist_constant = focal_length * actual_height_of_reflective_tape
```

Comment:
One last thing I forgot to mention: All of these distances are physical length units (mm/in etc). The size of the rectangle on your image is in pixels. There is one last conversion from pixels to physical length, which can be done with the physical sensor size of your camera. That might be hard to find online, but you can directly measure it, or design some experiments to derive it empirically.